packages/markuplint/src/api/ml-engine.ts

```diff
@@ -211,12 +211,10 @@ export default class MLEngine extends Emitter<MLEngineEventMap> {
 
 		// Exclude
 		const excludeFiles = configSet.config.excludeFiles ?? [];
-		for (const excludeFile of excludeFiles) {
-			if (this.#file.matches(excludeFile)) {
-				this.emit('exclude', this.#file.path, excludeFile);
-				fileLog('Excludes the file: %s', this.#file.path);
-				return null;
-			}
+		if (this.#file.ignored(excludeFiles)) {
+			// this.emit('exclude', this.#file.path, excludeFile);
```

Comment:
Is this comment necessary?
Seemed like you could delete it if it is not needed.

Reply:
Thanks, I'll remove it.